bench/benches/i386.rs

```diff
@@ -171,10 +172,16 @@ criterion_group!(
     config = Criterion::default().with_measurement(WallTime);
     targets = search_short_haystack, search_long_haystack
 );
+
+#[cfg(target_os = "linux")]
 criterion_group!(
     name = i386_perf_instructions;
     config = Criterion::default().with_measurement(PerfMeasurement::new(PerfMode::Instructions));
     targets = search_short_haystack, search_long_haystack
 );
 
+#[cfg(target_os = "linux")]
 criterion_main!(i386_wall_time, i386_perf_instructions);
+
+#[cfg(not(target_os = "linux"))]
+criterion_main!(i386_wall_time);
```

Comment:
I think it's ok to keep the wall time benchmarks on linux too. It's not because they are unreliable on my machines that they are everywhere ;)

Reply:
I didn't mean to take these away 😆  Actually they're still included for linux targets on line 184. It's a slightly confusing way to write I know but it's because unfortunately I can't put the `cfg` attribute within the `criterion_main` macro so I've had to repeat it.